js/turtleactions/IntervalsActions.js

```diff
@@ -19,6 +19,12 @@
  * Action methods are in camelCase.
 */
 
+/*exported setupIntervalsActions*/
+
+/* global MUSICALMODES, _, getNote, Singer, getModeLength, logo, NOINPUTERRORMSG, blocks, 
+MusicBlocks, Mouse, args
```

Comment:
are you sure `args` is a global here? where does it come from? Usually args contains the arguments for a block passed into the flow function. Asking as it seems to be a reason for a few bugs in some of the functions in this file.

Reply:
yes it is global:- please use this [link ](https://github.com/sugarlabs/musicblocks/blob/d081fe3ef8e4ef14c503a271dd7b417f8802ac7c/js/js-export/API/MeterBlocksAPI.js#L35)for your perusal

Reply:
@ricknjacky For the link you have shared, I think `args` is local to the `setBPM` function. Almost all functions in the `MeterBlockAPI` class are declaring local variable `args` in them. Also, I think all globals could be accessed in the browser console which args cannot. Also, in the `intervalActions.js` file there is only one instance of `args` in the `defineMode` function which I think is an error as when you try to use the blocks that make use of that function you get a console error about it. I have also opened a PR regarding it. cc - @meganindya

Reply:
Where is `args`?